firebase.json

```diff
@@ -145,6 +145,7 @@
       { "source": "/go/non-promo-write", "destination": "/tools/non-promotion-reasons#write", "type": 301 },
       { "source": "/go/null-safety-migration", "destination": "/null-safety/migration-guide", "type": 301 },
       { "source": "/go/pubignore", "destination": "/tools/pub/publishing#what-files-are-published", "type": 301 },
+      { "source": "/go/pubretract", "destination": "/tools/pub/publishing#retract", "type": 301 },
```

Comment:
```suggestion
      { "source": "/go/package-retraction", "destination": "/tools/pub/publishing#retract", "type": 301 },
```

@szakarias thoughts?

Reply:
I also like this better, so it does not get confused with a pub command

Reply:
I like the clearer URL. Can we use it, @szakarias?